### PR TITLE
APPSERV-124 Adds sections on roles and page sync, settings image update

### PR DESCRIPTION
**Pull request by jbee**
Updates in connection with https://github.com/payara/Payara/pull/4666 as well as some typos.

### CODE REVIEW

File: docs/modules/ROOT/pages/documentation/user-guides/monitoring/monitoring-console.adoc [hunk 1]
```diff
@@ -92,6 +91,63 @@ WARNING: The reset will affect all widgets and the general page settings of the
...
+[[monitoring-console-page-sync]]
+==== Page Synchronisation
+Monitoring Console uses the browser's local storage to store the page configuration (which includes the widgets placed on a page).
+This means by default changes to page configurations do not affect other users.
+To make pages available to others and to start from a common base configuration pages are explicitly synchronised with a persistent server page configuration.
+
+The chosen role controls the behaviour and available options for the page synchronisation.
+It is the _Administrator_'s role that is meant to manage the persistent server page configuration.
+The _Administrator_ pushes local changes to the server to update the base configuration.
+_User_s and _Guest_s pull the server configuration to update their local configuration with the base.
```
Comment (Pandrex247) [inline docs/modules/ROOT/pages/documentation/user-guides/monitoring/monitoring-console.adoc:103]: I'm not sure if it's just the GitHub preview, but it doesn't like this italics syntax despite working elsewhere. Best guess is the no space between the closing `_` and the `s`

File: docs/modules/ROOT/pages/documentation/user-guides/monitoring/monitoring-console.adoc [hunk 2]
```diff
@@ -92,6 +91,63 @@ WARNING: The reset will affect all widgets and the general page settings of the
...
+* _Guest_: by default updates all pages available remotely unless they are already up to date
+* _Administrator_ and _User_: by default updates all pages available remotely unless a page has been locally changed
+
+The newest version is marked by a green background for easier orientation.
+
+This dialogue can also be opened from the _Global_ settings by clicking the _Update Local Pages_ button shown _Page Sync_ property.
+
+image::monitoring-console/mc_setting_global_rolesync.png[Global Page Synchronisation Settings]
+
+_Administrator_s also find a button to _Update Remote Pages_ with any changes applied to local changes.
```
Comment (Pandrex247) [inline docs/modules/ROOT/pages/documentation/user-guides/monitoring/monitoring-console.adoc:121]: Again, italics are going weird here. Might just be GitHub preview.

File: docs/modules/ROOT/pages/documentation/user-guides/monitoring/monitoring-console.adoc [hunk 3]
```diff
@@ -92,6 +91,63 @@ WARNING: The reset will affect all widgets and the general page settings of the
...
+This is only needed for pages that are not using _auto_matic updates.
+
+To share a page that so far is not part of the persistent remote configuration open the _Page_ settings and look for the _Sync_ property:
+
+image::monitoring-console/mc_setting_page_sync.png[Page Synchronisation Settings]
+
+_User_s and _Administrator_s can both _Push_ a page to update the persistent remote configuration and _Pull_ the remote configuration to update the local page.
+_Administrator_s can also enable or disable the _auto_matic page synchronisation.
```
Comment (Pandrex247) [inline docs/modules/ROOT/pages/documentation/user-guides/monitoring/monitoring-console.adoc:129]: See above.

File: docs/modules/ROOT/pages/documentation/user-guides/monitoring/monitoring-console.adoc [hunk 4]
```diff
@@ -92,6 +91,63 @@ WARNING: The reset will affect all widgets and the general page settings of the
...
+
+[[monitoring-console-user-role]]
+==== Changing User Role
+The monitoring console can be used in one of three roles:
+
+* Guest: A user that uses the console only on occasion to check existing metrics on existing pages ("read-only" user).
+* User: A user that uses the console to experiment with its features without affecting others.
+* Administrator:  A user that uses the console to configure the configuration that is the basis for other users.
+
+When no role is set for the session the user is asked to chose the role when opening monitoring console.
```
Comment (Pandrex247) [inline docs/modules/ROOT/pages/documentation/user-guides/monitoring/monitoring-console.adoc:141]:
```suggestion
When no role is set for the session the user is asked to choose a role when opening the monitoring console.
```

File: docs/modules/ROOT/pages/documentation/user-guides/monitoring/monitoring-console.adoc [hunk 5]
```diff
@@ -278,8 +334,23 @@ This will then match all existing series that match the pattern. This feature ca
 
 [[monitoring-console-settings]]
 === Global Configurations
-Currently the configuration that controls the monitoring console UI is only stored in the browser's local storage.
-It can be exported and imported in the form of JSON files to share or distribute configurations.
+The configuration that controls the monitoring console UI.
+It is stored in the browser's local storage.
+For manual sharing the configuration can be exported and imported in the form of JSON files to share or distribute configurations.
```
Comment (Pandrex247) [inline docs/modules/ROOT/pages/documentation/user-guides/monitoring/monitoring-console.adoc:339]:
```suggestion
For manual sharing the configuration can be exported and imported in the form of JSON files.
```

File: docs/modules/ROOT/pages/documentation/user-guides/monitoring/monitoring-console.adoc [hunk 6]
```diff
@@ -278,8 +334,23 @@ This will then match all existing series that match the pattern. This feature ca
...
+Server integrated sharing can be done using the build in page synchronisation which is a two way update mechanism to update either the local configuration with server configuration or the other way around which allows to share changes made by one user with other users in a more convenient way.
```
Comment (Pandrex247) [inline docs/modules/ROOT/pages/documentation/user-guides/monitoring/monitoring-console.adoc:340]:
```suggestion
Server integrated sharing can be done using the built in page synchronisation which is a two-way update mechanism to update either the local configuration with server configuration or the other way around which allows the sharing of changes made by one user with other users in a more convenient way.
```

File: docs/modules/ROOT/pages/documentation/user-guides/monitoring/monitoring-console.adoc [hunk 7]
```diff
@@ -490,16 +561,16 @@ image::monitoring-console/mc_page_health-checks.png[Health Checks Page]
...
 * _Live Threads_: Total number of threads used by the JVM (including daemon threads)
 * _Daemon Threads_: Total number of daemon threads used by the JVM
 
-WARNING: Be aware that the number shown in the example image are not characteristic. For demontration purposes the thresholds have been set uncharacteristically low to be able to "force" entries. Usually the threshold for stuck threads should be in region of some seconds to a few minutes. The threshold for hogging threads should be well above 90% with a few reties.
+WARNING: Be aware that the number shown in the example image are not characteristic. For demonstration purposes the thresholds have been set uncharacteristically low to be able to "force" entries. Usually the threshold for stuck threads should be in region of some seconds to a few minutes. The threshold for hogging threads should be well above 90% with a few reties.
```
Comment (Pandrex247) [inline docs/modules/ROOT/pages/documentation/user-guides/monitoring/monitoring-console.adoc:573]:
```suggestion
WARNING: Be aware that the number shown in the example image are not characteristic. For demonstration purposes the thresholds have been set uncharacteristically low to be able to "force" entries. Usually the threshold for stuck threads should be in region of some seconds to a few minutes. The threshold for hogging threads should be well above 90% with a few retries.
```

File: docs/modules/ROOT/pages/documentation/user-guides/monitoring/monitoring-console.adoc (hunk 1 above)
Comment (OndroMih) [reply docs/modules/ROOT/pages/documentation/user-guides/monitoring/monitoring-console.adoc:103]: You're right, @Pandrex247, it now looks like this in Antora: image

@jbee, if you need to format only a part of a word, you need to double the underscores. This is called "unconstrained" formatting in the Asciidoc documentation: https://asciidoctor.org/docs/asciidoc-writers-guide/#quoted-text

This should work as expected:

```suggestion
__User__s and __Guest__s pull the server configuration to update their local configuration with the base.
```

File: docs/modules/ROOT/pages/documentation/user-guides/monitoring/monitoring-console.adoc (hunk 2 above)
Comment (OndroMih) [reply docs/modules/ROOT/pages/documentation/user-guides/monitoring/monitoring-console.adoc:121]:
```suggestion
__Administrator__s also find a button to _Update Remote Pages_ with any changes applied to local changes.
```

File: docs/modules/ROOT/pages/documentation/user-guides/monitoring/monitoring-console.adoc (hunk 3 above)
Comment (OndroMih) [reply docs/modules/ROOT/pages/documentation/user-guides/monitoring/monitoring-console.adoc:129]:
```suggestion
__User__s and __Administrator__s can both _Push_ a page to update the persistent remote configuration and _Pull_ the remote configuration to update the local page.
__Administrator__s can also enable or disable the __auto__matic page synchronisation.
```

**Comment by jbee** [comment]
Thanks @Pandrex247 and @OndroMih

**Pull request merged**